db/migrate/20200910051633_change_uploads_verified_to_integer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class ChangeUploadsVerifiedToInteger < ActiveRecord::Migration[6.0]
+  def up
+    add_column :uploads, :verification_status, :integer, null: false, default: 1, index: true
```

Comment:
Since we're doing a mass update to the column, we should leave out the index first so that the `UPDATE` below will be faster. Some sites may have a ton of uploads so the `UPDATE` may be potentially slow.

Reply:
Great point, will add the index after update.